fastlane/lib/fastlane/actions/setup_jenkins.rb

```diff
@@ -40,6 +43,9 @@ def self.run(params)
             add_to_search_list: params[:add_keychain_to_search_list],
             set_default: params[:set_default_keychain]
           )
+          ENV['MATCH_KEYCHAIN_NAME'] = keychain_path
+          ENV['MATCH_KEYCHAIN_PASSWORD'] = params[:keychain_password]
+          ENV["MATCH_READONLY"] = true.to_s
```

Comment:
```suggestion
          ENV['MATCH_KEYCHAIN_NAME'] ||= keychain_path
          ENV['MATCH_KEYCHAIN_PASSWORD'] ||= params[:keychain_password]
          ENV["MATCH_READONLY"] ||= true.to_s
```

I think we may went to only set these if they haven’t been set already by the user 🤔 Thoughts?

Reply:
Sure, will change it.

Reply:
argh, I am not up to date with those "newer" GitHub features, day to day I use BitBucket. I just noticed I could have just applied it. Anyway, did the same except, I left ENV["MATCH_READONLY"] ||= "true". Let me know, I can change it.